client/src/components/Payment/PaymentListForm/PaymentListForm.tsx

```diff
@@ -0,0 +1,34 @@
+import { useEffect, useState } from 'react';
+import Grid from '@material-ui/core/Grid';
+import Box from '@material-ui/core/Box';
+import PaymentCard from '../PaymentCard/PaymentCard';
+import ModalButton from '../../Modal/ModalButton/ModalButton';
+
+export default function PaymentListForm(): JSX.Element {
+  const [listPaymentMethods, setListPaymentMethods] = useState(<div>Loading...</div>);
+  useEffect(() => {
+    (async function () {
+      const res = await fetch('/payment/list-payment-methods', {
+        method: 'GET',
+      });
+      const data = await res.json();
+      setListPaymentMethods(
+        <>
+          <Box p={3} alignSelf="center">
+            <Grid container justify="center" spacing={2} xs={12}>
+              {data.paymentMethods.data.map((element: any) => (
+                <Grid item key={element.id}>
+                  <PaymentCard card={element.card} />
+                </Grid>
+              )) || 'You have no cards'}
+            </Grid>
+          </Box>
+          <Box textAlign="center">
+            <ModalButton>Add a new card</ModalButton>
+          </Box>
+        </>,
+      );
```

Comment:
Same comments here. It could also be a good idea to refactor this logic into a reusable component but since we are nearing the end of the project we can hold this off for another time 👍🏼